projects/addon-mobile/components/sheet-dialog/sheet-dialog.module.ts

```diff
@@ -0,0 +1,24 @@
+import {CommonModule} from '@angular/common';
+import {NgModule} from '@angular/core';
+import {IntersectionObserverModule} from '@ng-web-apis/intersection-observer';
+import {tuiAsDialog, TuiClickOutsideModule} from '@taiga-ui/cdk';
```

Comment:
`tuiAsDialog` need to backport from 3.x?

Reply:
No, let's just do it manually here.